pkg/scyllaclient/client_ping.go

```diff
@@ -32,7 +33,7 @@ func (c *Client) GetLiveNodesWithLocationAccess(ctx context.Context, nodes NodeS
 	err = parallel.Run(len(liveNodes), parallel.NoLimit, func(i int) error {
 		n := liveNodes[i]
 
-		if err := c.RcloneCheckPermissions(ctx, n.Addr, remotePath); err == nil {
+		if err = c.RcloneCheckPermissions(ctx, n.Addr, remotePath); err == nil {
```

Comment:
Why this change ? `err:=` defines scope of current `err` variable to the method where it's created.

Reply:
I changed that because after the if I am still using this error in `nodeErr[i] = err`, but i guess that I should just assign the result of this operation to error outside of if statement, so it looks cleaner.